resources/schema/USDM.yaml

```diff
@@ -0,0 +1,67 @@
+"`this`": "null"
+ACTIVITY: Activity
```

Comment:
I didn't worry too much about the mapping file because I knew this would change when we have a good sample study for 2.6 and go away altogether with instancetype.
I made the mapping changes you identified.

Reply:
Updated